tests/400-serlo-editor.ts

```diff
@@ -89,3 +89,290 @@ Scenario('Markdown list shortcut', async ({ I }) => {
 
   I.seeElement('ul[data-slate-node="element"]')
 })
+
+Scenario('Plugin Suggestions Menu', async ({ I }) => {
+  I.amOnPage('/entity/create/Article/1377')
+
+  //Upload first image
+
+  I.click("input[placeholder='https://example.com/image.png']")
+
+  I.type('https://denkspace.de/zukunftswerk.gif')
```

Comment:
probably better to use images that are hosted on serlo.org here, otherwise tests might fail because of other servers.